std_msgs/Header header

uint32 class_id
string class_name
float32 probability
sensor_msgs/RegionOfInterest bounding_box
